spec/features/families/admin_views_all_families_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+feature "Admin can see all the families when in families#index" do
+  context "when click on the 'All' button" do
```

Comment:
I think this needs to be scenario not context?

Reply:
Oh, thanks, I was getting a little bit confused trying to figure out what to use!

Reply:
Done with fixup in `4d01de2`